index.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+Nowa strona
```

Comment:
Zmień "Nowa strona" na "Stara strona". Takimi komentarzami jestem w stanie wskazywać Ci dokładnie, w której linijce masz błąd. Ta czynność nazywa się "code review" i właśnie dostałeś pierwszy code review. :) W momencie gdy zmienisz to, dam Ci approve na ten Pull Request i będziesz mógł zmergować to do głównej gałęzi (master).